Source/Initialization/DivCleaner/ProjectionDivCleaner.H

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2024 Helion Energy, Inc.
+ *
+ * Authors: S. Eric Clark
+ * License: None
+ */
+#pragma once
```

Comment:
Nice! We might want to start a coordinated effort to switch all WarpX header files to this approach instead of the old, lengthier 
```
#ifndef _MYHEADER_H_
#define _MYHEADER_H_
...
#endif
```

Reply:
The `#pragma once` looks like a nice feature but is not standard. This should be checked is its appropriate here. Also, the header should follow the same pattern as other files, specifically "License: BSD-3-Clause-LBNL".

Reply:
I think the pragma once is pretty widely supported. I switched over to include guards to be consistent with the rest of the header files.